notebooks/Notebook2.md

```diff
@@ -2,33 +2,56 @@
-improve the project? Did the feedback point out or offer something you hadn't considered?
-Did it help you make a design decision? Was it helpful in addressing the most pressing
-issues in your project? How will you incorporate the feedback into your work? Will you
-change something about the design, implementation, or evaluation as a result?
+“I think it would be useful to ensure you have some documentation of this structure both for: 1) your own reference, which may allow you to reflect on what other use cases you may want in your design and 2) users writing in your DSL for the first time, it will allow them to have a template (and some example calls would be good) to look at when they are learning to write out commands in your DSL.”
+
+“Let me know if you want to talk about the repo at all, or if there is something specific I can look at.”-Prof Ben
+
+“I liked that you noted that the work was tedious despite not being complex, is there any way that you think you could have implemented these that would have been less tedious?”
+
+I will address the three critiques above:
+
+I agree. In the end I think that I will need some form of documentation for the language that shows different possibilities for each input group of the language as well as the general structure. I think I will plan to work on that near the end of the project term. Right now I want to focus on other aspects. 
+
+To Prof Ben, I do think I may need some help with my repository. Although It has been working pretty well I ran into a problem this week when trying to implement a small interface for inputting my language. It turns out that codespaces does not know how to build a physical java application type interface without the proper setup in the repository or connections to a remote explorer. I would appreciate help with that sometime next week. 
+
+Sadly I do not think I can simplify the tedious work that I had to do much more. You will see that there were some changes to the files for the player types this week. The changes allowed for simplification of various parts of the player types such as getting rid of the name aspect or adding simpler setters for each type. 
+
 
 ## Description
 
-**TODO:** Fill in this part with information about your work this week:
-important design decisions, changes to previous decisions, open questions,
-exciting milestones, preliminary results, etc. Feel free to include images
-(e.g., a sketch of the design or a screenshot of a running program), links to
-code, and any other resources that you think will help clearly convey your
-design process.
+This week I worked on a few main things such as: building the bride from input to data structure through a parser, Implementing some sort of single line input either via a physical java interface or that being the command line, and looking over and simplifying the player type classes with individual setters per input type vs one big setter that sets all input types. 
```

Comment:
Great idea to include an overview of the work you did for the week at the top of the description. I don't think I did but it is very useful before getting into very technical details to understand what part of the project you were addressing